packages/core/test/icon/iconTests.tsx

```diff
@@ -24,7 +24,8 @@ describe("<Icon>", () => {
 
     it("renders iconName class", () => assertIcon(<Icon iconName="calendar" />, "calendar"));
 
-    it("supports prefixed iconName", () => assertIcon(<Icon iconName={IconClasses.AIRPLANE} />, "airplane"));
+    // uncomment for compiler error
+    // it("does not support prefixed iconName", () => assertIcon(<Icon iconName="pt-icon-airplane" />, "airplane"));
```

Comment:
with TS 2.6 you can use `// @ts-ignore` to ignore the compiler error. then you could make this test verify that it doesn't render anything

Reply:
_whoa_